src/transformers/modeling_flax_outputs.py

```diff
@@ -0,0 +1,239 @@
+# Copyright 2021 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from dataclasses import dataclass
+from typing import Optional, Tuple
+
+from jaxlib.xla_extension import DeviceArray
+
+from .file_utils import ModelOutput
+
+
+@dataclass
+class FlaxBaseModelOutput(ModelOutput):
+    """
+    Base class for model's outputs, with potential hidden states and attentions.
+
+    Args:
+        last_hidden_state (:obj:`DeviceArray` of shape :obj:`(batch_size, sequence_length, hidden_size)`):
```

Comment:
We usually say `torch.Tensor` or `tf.Tensor`, so it  would be nice to have a full path. At the same time, `jaxlib.xla_extension.DeviceArray` is a mouthful. Is there a common abbreviation of `jaxlib.xla_extension` we could use?